### ISSUE TITLE
#183 "Don't allow divide by zero" commit causing problems

### ISSUE THREAD

**Issue by pijyoi** [pyqtgraph/pyqtgraph#214]
The commit made in #183 is causing problems in my image display code.
I am displaying a large image, thus need to enable auto-downsampling.
After #183, the auto-downsampling doesn't seem to be kicking in.

```python
imgitem = pyqtgraph.ImageItem(data, autoDownsample=True)
imv = pyqtgraph.ImageView(imageItem=imgitem)
imv.show()
```

Before #183:
```python
xds = max(1, int(1/w))  # 51
yds = max(1, int(1/h))  # 51
```

After #183:
```python
xds = int(1/max(1,w))  # 1
yds = int(1/max(1,h))  # 1
```

The w and h values in my application are around 0.0192

**Comment by pijyoi** [comment]
fixed by #369

**Comment by campagnola** [comment]
Thanks @pijyoi !
